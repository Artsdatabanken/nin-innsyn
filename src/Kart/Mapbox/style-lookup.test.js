import stylelookup from './style-lookup'

const map = {
  getLayer: function(layer) {
    return null
  },
}

test('lag stiler', () => {
  expect(stylelookup(map, 'NA_T')).toMatchSnapshot()
})
