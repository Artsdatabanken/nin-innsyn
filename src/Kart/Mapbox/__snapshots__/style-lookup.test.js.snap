// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stil MI_KA 1`] = `
Object {
  "id": "kalk",
  "paint": Object {
    "fill-color": Object {
      "base": 1,
      "default": "hsl(0, 4%, 94%)",
      "property": "KALKINNHOLD_HOVEDBERGART",
      "stops": Array [
        Array [
          0,
          "hsla(0, 0%, 0%, 0)",
        ],
        Array [
          1,
          "hsla(0, 6%, 94%, 0.6)",
        ],
        Array [
          2,
          "hsla(0, 19%, 88%, 0.6)",
        ],
        Array [
          3,
          "hsla(0, 35%, 80%, 0.6)",
        ],
        Array [
          4,
          "hsla(0, 59%, 63%, 0.6)",
        ],
        Array [
          5,
          "hsla(0, 84%, 32%, 0.6)",
        ],
      ],
      "type": "exponential",
    },
  },
  "source": "composite",
  "source-layer": "kalk",
  "type": "fill",
}
`;

exports[`stil NA 1`] = `
Object {
  "filter": Array [
    "has",
    "NA_T",
  ],
  "id": "NA_T",
  "interactive": true,
  "paint": Object {
    "fill-color": "#FFFF00",
    "fill-opacity": 0.13,
    "fill-outline-color": "#FF0000",
  },
  "source": "composite",
  "source-layer": "naturomrader6",
  "type": "fill",
}
`;
